src/task_list/moduls.py

```diff
@@ -0,0 +1,146 @@
+import json
+import pathlib
+
+
+class DataJson:
+    'Класс для работы с файлом'
+
+    def __init__(self):
+        self.file = pathlib.Path('task_list/src/config/data.json').resolve()
+
+
+    def read_data_json(self):    # Чтение файла с задачами
+        with open(self.file, 'r', encoding='utf-8') as file:
+            tasks = json.load(file)
+
+        return tasks
+
+
+    def write_to_file(self, task):    # Запись задач в файл
```

Comment:
Про описание думаю тут ты понял

типизации тоже не хватает 
```suggestion
    def write_to_file(self, task: dict) -> None:    # Запись задач в файл
```

если не перепутал, то это словарь